docs/rules/consistent-return.md

```diff
@@ -21,6 +21,10 @@ Here, one branch of the function returns `true`, a Boolean value, while the othe
 
 This rule is aimed at ensuring all `return` statements either specify a value or don't specify a value.
 
+It excludes constructors which, when invoked with the `new` operator, return the instantiated object if another object is not explicitly returned.  This rule judges from following conditions whether or not the function is a constructor:
```

Comment:
I think this should be changed slightly. `following conditions` imply more then one.